pkgs/development/python-modules/gudhi/default.nix

```diff
@@ -43,13 +42,22 @@ buildPythonPackage rec {
     "-DPython_ADDITIONAL_VERSIONS=3"
   ];
 
+  prePatch = ''
+    substituteInPlace src/python/CMakeLists.txt \
+      --replace '"''${GUDHI_PYTHON_PATH_ENV}"' ""
+  '';
+
   preBuild = ''
     cd src/python
   '';
```

Comment:
Usually we set 
```suggestion
  sourceRoot = "${src.name}/src/python";
```

Reply:
The build system depends on `CMakeLists.txt` in the root of the project, so unfortunately custom `sourceRoot` breaks the build.

Reply:
OK.